README.md

```diff
@@ -10,10 +10,10 @@ Uranium is released under the terms of the LGPLv3 or higher. A copy of this lice
 
 Dependencies
 ------------
-- Qt 5 with QtQuick 2, QtQuick Controls
+- [Qt 5](https://doc.qt.io/qt-5/) with [QtQuick 2](https://doc.qt.io/archives/qt-5.11/qtquickcontrols2-index.html), [QtQuick Controls](https://doc.qt.io/qt-5/qtquickcontrols-index.html)
 - Python 3, with the following modules
-  - PyQt5
-  - Numpy
-  - Arcus
-  - Shapely
-  - Scipy
\ No newline at end of file
+  - [PyQt5](https://github.com/baoboa/pyqt5)
+  - [Numpy](https://github.com/numpy/numpy)
+  - [Arcus](https://github.com/cuenca-mx/arcus-python)
```

Comment:
This is not the correct Arcus. This is the Arcus we're intending: https://github.com/Ultimaker/libArcus